#### **TASK**

Given an integer, , and  space-separated integers as input, create a tuple, , of those  integers. Then compute and print the result of hash().

<u>Note</u>: *hash()* is one of the functions in the __builtins__ module, so it need not be imported.

**Input Format**
The first line contains an integer, , denoting the number of elements in the tuple.
The second line contains  space-separated integers describing the elements in tuple .

**Output Format**
Print the result of hash().

**Sample Input 0**
<p>2</p>
1 2

In [1]:
n = int(input())
integer_list = map(int, input().split())
t=tuple(integer_list)
print(hash(t))

-3550055125485641917


***DISCLAIMER:***

Para la página de Hacker Rank, la respuesta era 3713081631934410656 y no -3550055125485641917 como sale aquí. Esto ocurre por la diferencia en las versiones del KERNEL de PYTHON, el mío que imprime el 2do valor es con Python 3.11.4; en cambio, el primer valor es con Pypy 3.